### PR TITLE
chore: No-op Packer, useful for testing.

### PR DESCRIPTION
**Pull request by Moopli**
Signed-off-by: Filip Burlacu <filip.burlacu@securekey.com>

### CODE REVIEW

**Comment by troyronda** [comment]
@Moopli Where do you plan to use the no-op packer?

File: go.mod [hunk 1]
```diff
@@ -10,6 +10,7 @@ require (
 	github.com/golang/mock v1.3.1
 	github.com/google/uuid v1.1.1
 	github.com/gorilla/mux v1.7.3
+	github.com/mr-tron/base58 v1.1.2
```
Comment (troyronda) [inline go.mod]: We are using a different base58 package elsewhere.

Comment (troyronda) [reply go.mod]: "github.com/btcsuite/btcutil/base58"

Comment (Moopli) [reply go.mod]: Good catch, thought I'd gotten them all.

Comment (Moopli) [reply go.mod]: Ah, I did get them all, but I didn't tidy the mod file.

**Comment by Moopli** [comment]
@troyronda it's for testing, to make tests results easier to parse - I've moved it to mock since it's not intended for prod.

**Pull request merged**